### DISCUSSION TITLE
Ctrl-to-Copy for Schedule helpers

### DISCUSSION THREAD

**Discussion by peter1rhodes** [home-assistant/frontend#21912] [Other feature requests]
When creating a Schedule helper, it is possible to drag and drop a period to a different day. It would be very useful if this behaviour became a Copy action when the Control key is being held, for example (or another convenient shortcut). This way, the user could very quickly and intuitively copy time blocks to multiple days, which would cover the most common use of Schedule helpers.

![image](https://github.com/user-attachments/assets/1417695e-046b-418e-abf3-5ef14cf0d9d3)

**Comment by Marvin-D-Martian** [comment]
Absolutely. I actually tried holding down different combinations of keys expecting for it to work. Makes perfect sense to implement that. Most of the schedules repeat for several days and having it "ctrl-cop-drag" would make it perfect. Pretty please?
